app/uk/gov/hmrc/alcoholdutycontactpreferences/services/EventHubBounceService.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.alcoholdutycontactpreferences.services
+
+import cats.data.EitherT
+import play.api.Logging
+import play.api.http.Status.BAD_REQUEST
+import uk.gov.hmrc.alcoholdutycontactpreferences.config.AppConfig
+import uk.gov.hmrc.alcoholdutycontactpreferences.connectors.SubmitPreferencesConnector
+import uk.gov.hmrc.alcoholdutycontactpreferences.models.{EventDetails, PaperlessPreferenceSubmission, PaperlessPreferenceSubmittedResponse}
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.bootstrap.http.ErrorResponse
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class EventHubBounceService @Inject() (
+  submitPreferencesConnector: SubmitPreferencesConnector,
+  appConfig: AppConfig
+)(implicit ec: ExecutionContext)
+    extends Logging {
+
+  def handleBouncedEmail(
+    eventDetails: EventDetails
+  )(implicit hc: HeaderCarrier): EitherT[Future, ErrorResponse, PaperlessPreferenceSubmittedResponse] =
+    getAppaIdFromEnrolmentString(eventDetails.enrolment) match {
+      case Right(appaId) =>
+        val contactPreferenceSubmission = PaperlessPreferenceSubmission(
+          paperlessPreference = false,
+          emailAddress = Some(eventDetails.emailAddress),
```

Comment:
@matthew-goh  - I am thinking as this is a bounced email event we should omit the email address and email verified fields? The business requirement here is simply to revert to paper and to set the bounced indicator for a given APPAID. As these fields are non mandatory I think they should be omitted to avoid any confusion in ETMP?